server/services/stats_service.py

```diff
@@ -114,7 +114,8 @@ def _set_counters_after_invalidated(task_id: int, project: Project, user: User):
     def get_latest_activity(project_id: int, page: int) -> ProjectActivityDTO:
         """ Gets all the activity on a project """
 
-        results = db.session.query(TaskHistory.action, TaskHistory.action_date, TaskHistory.action_text, User.username) \
+        results = db.session.query(TaskHistory.action, TaskHistory.action_date, TaskHistory.action_text,
```

Comment:
Thanks, @ethan-nelson. Checking this PR, it seems has been fixed in the meanwhile by @ramyaragupathy and is in [current `develop`](https://github.com/hotosm/tasking-manager/blob/develop/server/services/stats_service.py#L128). coming from the PR #1344 (see [diff](https://github.com/hotosm/tasking-manager/pull/1344/files#diff-52db8babc6b474fbcba619e6b9a0cdfaR128)).

```
        results = db.session.query(
                TaskHistory.id, TaskHistory.task_id, TaskHistory.action, TaskHistory.action_date,
                TaskHistory.action_text, User.username
            ).join(User).filter(
                TaskHistory.project_id == project_id,
                TaskHistory.action != 'COMMENT'
            ).order_by(
                TaskHistory.action_date.desc()
            ).paginate(page, 10, True)
```

Please close the PR in case you can confirm.